app/models/container-lease.ts

```diff
@@ -0,0 +1,34 @@
+import { Model, Prop, Record } from "app/core";
+
+export interface ContainerLeaseAttributes {
+    status: LeaseStatus;
+    state: LeaseState;
+    duration: string;
+}
+
+/**
+ * Class for displaying blob container lease information.
+ */
+@Model()
+export class ContainerLease extends Record<ContainerLeaseAttributes> {
+    @Prop() public status: LeaseStatus;
+    @Prop() public state: LeaseState;
+    @Prop() public duration: string;
+}
+
+export type LeaseStatus = "locked" | "unlocked" | "unspecified";
```

Comment:
just done the container ones for now, will fix up all the rest of the enums at some stage.